networking/amplify/backend/api/mobile/resolvers/Mutation.createTeam.auth.1.req.vtl

```diff
@@ -7,12 +7,12 @@ $util.qr($ctx.stash.put("hasAuth", true))
 $util.unauthorized()
 #end
```

Comment:
I think there is a bug with Amplify that requires us to manually set these resolvers. Pretty annoying so I'll talk to our AWS contact next time. See Wiki for a link to an Amplify issue that confirms this and recommends the below edits to resolve